src/Microsoft.Extensions.CodeGeneration.Utils/project.json

```diff
@@ -6,17 +6,24 @@
   "dependencies": {
     "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0-*",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
-    "System.AppContext": "4.1.0-*",
-    "System.Security.Cryptography.Algorithms": "4.1.0-*",
     "System.Threading.Tasks.Parallel": "4.0.1-*"
   },
   "frameworks": {
+    "net451": {
+        "dependencies": {
+            "System.Threading.Tasks":"4.0.0-*"
```

Comment:
Its needed for the compilation.Emit call [here](https://github.com/aspnet/Scaffolding/blob/pb/net45support/src/Microsoft.Extensions.CodeGeneration.Utils/CommonUtilities.cs#L22 ). The Emit method has a default parameter of type `CancellationToken` which is defined in the System.Threading.Tasks assembly. 
The code doesn't build without this.